### PR TITLE
fixes #125 - Created a new category for the varous loklak API libraries

### LINKED ISSUE #125: Creating a new category for APIs
Loklak has presently got api implementations in python, go, php, ruby and nodejs, and I guess more will be developed in future. It would be nice if these libraries are showcased on apps.loklak.org and a separate category is made for them.
A rough outline of the category :
Clicking on the category will show all the desired tiles as we have for other apps.
Clicking on a specific tile will take the user to the store listing page where he/she will be provided with all the necessary information on how to use the library, all its features;
The page will also contain a link to the repository page so that user can browse the code and use the library.
In this way users will get aware of the various loklak libraries from apps.loklak.org itself

### PR DESCRIPTION
**Pull request by djmgit**
fixes issue #125. Python, Ruby, NodeJs, Go and Java library
for loklak are now showcased on apps.loklak.org.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
Test link : http://loklakapps.surge.sh/#APIlibraries

@kavithaenair @SKrPl @Achint08 open for review.

**Comment by kavithaenair** [comment]
@djmgit check travis

**Comment by Achint08** [comment]
@djmgit Not working as expected yet. Kindly checkout the link.

**Comment by djmgit** [comment]
@Achint08 can you please elaborate what is the problem :)

**Comment by Achint08** [comment]
@djmgit I don't know. But your demo link was not properly rendered.

File: loklakPythonApi/getStarted.md [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+##### Peers - API To find out the loklak peers
+
+Finding the list of loklak peers, use the object created above `l.peers()` which returns a json response containing all the peers connected to `loklak.org`
+
+##### Users API
+
+What this can do ?
+
+- Fetch the details of one user
+- Fetch the details of the user along with number of their followes and following
```
Comment (kavithaenair) [inline loklakPythonApi/getStarted.md]: Typo

**Comment by djmgit** [comment]
@kavithaenair documentations are present in the corresponding repos, however creating a separate category for libraries on apps.loklak.org itself will allow us to showcase the various use full libraries on this site itself, it will bring the various loklak components at one place, developers and users will every thing showcased at a single place, this was my reason behind creating this new category :)

**Comment by djmgit** [comment]
@Achint08 made changes
http://loklakapps.surge.sh/#LoklakLibraries

Presently in the header there is no space between "loklak" and "libraries", but that is due to separate problem, I will be solving it in a separate issue.

**Comment by djmgit** [comment]
@mariobehling I have changed the tile images

**Comment by mariobehling** [comment]
It is Loklak Libraries in the sidebar and LoklakLibraries in the menu.

**Comment by mariobehling** [comment]
The similar apps sidebar still scales images wrongly.

Please test the app in detail.
screenshot from 2017-06-17 19-15-08

**Comment by djmgit** [comment]
@mariobehling I have noticed both the issues which you mentioned but those are not caused by this PR, so I though of solving them as a separate issue, the space problem is caused by app_list.js and the images problem in the side bar is due to image container (details.css). Since they are not directly related to creation of library category so I have not included them in this PR. Should I update my PR or do it in next iteration?

**Comment by mariobehling** [comment]
If you create a new category, then I test all parts of that category. If some parts of that category don't work as expected, then they have to be solved.

We should start to practice for moving this project to a production stage officially. Please ensure that things are working as expected. Also please help to avoid "mini-issues" like LoklakLibraries instead of Loklak Libraries.

**Comment by djmgit** [comment]
@mariobehling sure, I will update my PR and make the required changes

**Comment by djmgit** [comment]
Test link : http://loklakapps.surge.sh/#LoklakLibraries

@mariobehling made the changes.

**Pull request merged**